cmd/dnet/dnet.go

```diff
@@ -106,7 +106,7 @@ type dnetConnection struct {
 }
 
 func (d *dnetConnection) dnetDaemon() error {
-	controller, err := libnetwork.New()
+	controller, err := libnetwork.New("")
```

Comment:
> Shouldn't this be trying to pass the default /etc/default/libnetwork.toml
`controller` defaults it to `/etc/default/libnetwork.toml` when the filename is ""

> Also shouldn't we add a config option for folks to add a different config file location
Yes. atm, we have `LIBNETWORK_CFG` env variable that folks can use to setup the file.
But yes, we can add a config flag in `dnet` daemon. good idea.
LIBNETWORK_CFG